easybuild/easyconfigs/x/XZ/XZ-5.2.5_compat-libs.patch

```diff
@@ -0,0 +1,35 @@
+Based on https://git.centos.org/rpms/xz/blob/c7/f/SOURCES/xz-5.2.2-compat-libs.patch
+(which is a CentOS7-only patch)
+but includes the XZ_5.2 symbols as the lack of these is causing problems in
+other scenarios
+(see https://github.com/easybuilders/easybuild-easyconfigs/issues/14991)
+
+diff -rupN xz-5.2.5/src/liblzma/liblzma.map xz-5.2.5_updated/src/liblzma/liblzma.map
+--- xz-5.2.5/src/liblzma/liblzma.map	2020-03-17 14:28:54.000000000 +0000
++++ xz-5.2.5_updated/src/liblzma/liblzma.map	2022-07-13 18:54:15.280407061 +0000
+@@ -95,14 +95,21 @@ global:
+ 	lzma_vli_size;
+ };
+ 
+-XZ_5.2 {
++XZ_5.1.2alpha {
++global:
++	lzma_stream_encoder_mt;
++	lzma_stream_encoder_mt_memusage;
++} XZ_5.0;
++
++XZ_5.2.2 {
+ global:
+ 	lzma_block_uncomp_encode;
+ 	lzma_cputhreads;
+ 	lzma_get_progress;
+-	lzma_stream_encoder_mt;
+-	lzma_stream_encoder_mt_memusage;
```

Comment:
The older patch did not remove these lines but then the linker throws some warnings about them already being declared in `XZ_5.1.2alpha`